src/start/AuthScreen.js

```diff
@@ -260,7 +260,7 @@ class AuthScreen extends PureComponent<Props> {
       id_token: credential.identityToken,
     });
 
-    openLink(`${this.props.realm}/complete/apple/?${params}`);
+    openLink(`${this.props.realm.toString()}/complete/apple/?${params}`);
```

Comment:
This probably wants to be a `URL` computation.

(That'd be a followup, another thing we can do to take advantage of having `URL` objects everywhere; for this branch a TODO comment would be fine.)